code/game/objects/structures/barsigns.dm

```diff
@@ -246,7 +246,7 @@
 /datum/barsign/emergencyrumparty
 	name = "The Emergency Rum Party"
 	icon = "emergencyrumparty"
-	desc = "Still serving drinks that were banned years ago."
+	desc = "Recently relicenced after a long closure."
```

Comment:
American spelling is standard throughout the codebase and game. Make it "relicensed."